### DISCUSSION TITLE
simple-obfuscation-of-email-addresses-using-javascript/

### DISCUSSION THREAD

**Discussion by andrewlock** [andrewlock/blog-comments#329] [Announcements]
# simple-obfuscation-of-email-addresses-using-javascript/

In this post I show a simple way to obfuscate email addresses to make it harder for bots to scrape them from your site, similar to Cloudflare Scrape Shield.

https://andrewlock.net/simple-obfuscation-of-email-addresses-using-javascript/

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at April 21 2020, 06:05, in reply to [khalidabuhakmeh](https://disqus.com/by/khalidabuhakmeh/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/329#discussioncomment-6058515)</em>

---

<p><a href="https://media1.giphy.com/media/wTTaiw8GECOK4/giphy.gif" rel="nofollow noopener" title="https://media1.giphy.com/media/wTTaiw8GECOK4/giphy.gif">https://media1.giphy.com/media/wTTaiw8GECOK4/giphy.gif</a></p>

[a comment by an automated account is not shown]

**Comment by salivity** [comment]
I created a tool to create a simple png image to protect your email addresses from spam bots. https://davidclews.com/article/283.html
